cli-output/src/cli_output.rs

```diff
@@ -1634,6 +1634,42 @@ impl fmt::Display for CliSignOnlyData {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CliSignOnlyDatas {
+    pub cli_sign_only_datas: Vec<CliSignOnlyData>,
+}
+
+impl fmt::Display for CliSignOnlyDatas {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        writeln!(f)?;
+        for data in &self.cli_sign_only_datas {
+            fmt::Display::fmt(data, f)?;
+        }
+        Ok(())
+    }
+}
+
+impl QuietDisplay for CliSignOnlyDatas {
+    fn write_str(&self, w: &mut dyn std::fmt::Write) -> std::fmt::Result {
+        writeln!(w)?;
+        for data in &self.cli_sign_only_datas {
+            QuietDisplay::write_str(data, w)?;
+        }
+        Ok(())
+    }
+}
+
+impl VerboseDisplay for CliSignOnlyDatas {
+    fn write_str(&self, w: &mut dyn std::fmt::Write) -> std::fmt::Result {
+        writeln!(w)?;
+        for data in &self.cli_sign_only_datas {
+            VerboseDisplay::write_str(data, w)?;
+        }
+        Ok(())
+    }
+}
+
```

Comment:
```suggestion
impl QuietDisplay for CliSignOnlyDatas {}
impl VerboseDisplay for CliSignOnlyDatas {}

```

Default impls already fallback to the `Display` impl